sktime/forecasting/model_evaluation/_functions.py

```diff
@@ -40,6 +40,69 @@ def _check_strategy(strategy):
         raise ValueError(f"`strategy` must be one of {valid_strategies}")
 
 
+def _check_scores(metrics) -> Dict:
+    """Validate and coerce Metric objects and segregate them based on predict type.
+
+    Parameters
+    ----------
+    metrics : sktime accepted metrics or a list of them or None
+
+    Return
+    ------
+    metrics_type : Dict
+        The key is metric types and its value is a list of its corresponding metrics.
+    """
+    if not isinstance(metrics, List):
+        metrics = [metrics]
+
+    metrics_type = {}
+    for metric in metrics:
+        metric = check_scoring(metric)  # validate and convert to sktime BaseMetric
+        # collect predict type
+        # I think there's no need to check for attribute "get_tag" as all metric class
+        # has that method, no? I dont see the need to do if here, did i miss anything?.
+        # if hasattr(metric, "get_tag"):
+        scitype = metric.get_tag(
+            "scitype:y_pred", raise_error=False, tag_value_default="pred"
+        )
+        # else:  # If no scitype exists then metric is a point forecast type
+        #     scitype = pred
+        if scitype not in metrics_type.keys():
+            metrics_type[scitype] = [metric]
+        else:
+            metrics_type[scitype].append(metric)
+    return metrics_type
+
+
+def _get_metada_for_dask(metric_types: Dict, return_data: bool, cutoff_dtype) -> Dict:
```

Comment:
is there a typo in the name of this function?

Reply:
indeed, it is.